src/jdk.jdwp.agent/share/native/libjdwp/transport.c

```diff
@@ -345,6 +352,24 @@ transport_waitForConnection(void)
     }
 }
 
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+void
+transport_waitForConnectionOnRestore(void)
+{
+    /*
+     * If suspended on VM restore, we also need to wait for a connection,
+     * since the VM won't continue without a remote debugger telling it to.
+     */
+    if (debugInit_suspendOnRestore()) {
```

Comment:
Suggestions elsewhere make this call to `debugInit_suspendOnRestore()` redundant; suggest it be removed.

Reply:
Removed this `debugInit_suspendOnRestore())` call.